docs/batch-changes/view-batch-changes.mdx

```diff
@@ -0,0 +1,33 @@
+# Viewing Batch Changes
+
+<p className="subtitle">Learn how to view, search, and filter your Batch Changes.</p>
+
+## Viewing batch changes
+
+You can view a list by clicking the **Batch Changes** icon in the top navigation bar:
+
+![batch-change-icon-to-view](https://sourcegraphstatic.com/docs/images/batch_changes/2024/batch_changes_icon_in_menu.png)
+
+### Title-based search
+
+You can search through your previously created batch changes by title. This search experience makes it much easier to find the batch change you're looking for, especially when you have large volumes of batch changes to monitor.
+
+Start typing the keywords that match the batch change's title, and you will see a list of relevant results.
+
+![title-based-search-batch-changes](https://storage.googleapis.com/sourcegraph-assets/Docs/title-based-search-bc.png)
+
+## Filtering Batch Changes
+
+You can also use the filters to switch between showing all open or closed Batch Changes.
```

Comment:
Nit: I'd remove "the" from "the filters" (and "the" from "the keywords" above) b/c it's just unnecessary to specify IMO